src/platform_impl/orbital/event_loop.rs

```diff
@@ -711,17 +710,15 @@ impl RootActiveEventLoop for ActiveEventLoop {
     }
 
     fn available_monitors(&self) -> Box<dyn Iterator<Item = crate::monitor::MonitorHandle>> {
-        let mut v = VecDeque::with_capacity(1);
-        v.push_back(crate::monitor::MonitorHandle { inner: MonitorHandle });
-        Box::new(v.into_iter())
+        Box::new(iter::empty())
     }
 
     fn system_theme(&self) -> Option<Theme> {
         None
     }
 
     fn primary_monitor(&self) -> Option<crate::monitor::MonitorHandle> {
-        Some(crate::monitor::MonitorHandle { inner: MonitorHandle })
+        None
     }
```

Comment:
Nit: I'd prefer to keep functional changes like these out of refactoring PRs in the future, but it's fine for now.

Though maybe add a changelog entry for it?

Reply:
It's not really functional iirc, since monitors are not supported.